test/index.js

```diff
@@ -12,6 +12,14 @@ describe('union type', function() {
     var Point = Type({Point: [isNumber, isNumber]});
     assert.equal('function', typeof Point.Point);
   });
+  it('type can be used with instanceof', function() {
+    var Point = Type({Point: [isNumber, isNumber]});
+    assert.equal(Point.Point(4,2) instanceof Point, true);
+  });
+  it('constructor can be used with instanceof', function() {
+    var Point = Type({Point: [isNumber, isNumber]});
+    assert.equal(Point.Point(4,2) instanceof Point.Point, true);
```

Comment:
Why should `Point.Point` be an instance of the constructor for a `Point`? This doesn't seem right to me. `Point` represents a type. But `Point.Point` is just a function that creates a value of the type.